libnetwork/resolver.go

```diff
@@ -393,6 +403,9 @@ func (r *Resolver) serveDNS(w dns.ResponseWriter, query *dns.Msg) {
 			}
 		}
 		resp.Truncate(maxSize)
+		span.AddEvent("found local record", trace.WithAttributes(
+			attribute.String("libnet.resolver.resp", resp.String()),
+		))
```

Comment:
Would it make sense to log it instead of creating the event explicitly, so it's also visible in logs without tracing?

Reply:
Good question! Actually I'm not sure whether it should be a log or a trace event, maybe something to discuss during a maintainers call.

Reply:
If using the standard logger it will be added to traces anyway.

Reply:
Yep, but on OTOH the daemon's `--log-level` will determine whether it'll be made available in the trace or not; and we don't have a way to set a log-level for a specific subsystem :thinking: 

Btw, there's a bug that make logs to be duplicated in traces.

Reply:
> Btw, there's a bug that make logs to be duplicated in traces right now.

I saw that! I wonder if that's because bklog is also modifying the standard logger.
I think with the intended logrus hook it should log all levels.
This is what we are using: https://github.com/containerd/containerd/blob/c5ce2a5f750ad89ab02974d047744517346578f3/tracing/log.go#L30-L58